src/frontend/komponenter/Felleskomponenter/Hendelsesoversikt/BrevModul/Brevskjema.tsx

```diff
@@ -435,6 +445,34 @@ const Brevskjema = ({ onSubmitSuccess }: IProps) => {
                             size={'small'}
                         />
                     )}
+                {skjema.felter.brevmal.verdi &&
+                    [
+                        Brevmal.VARSEL_OM_ÅRLIG_REVURDERING_EØS,
+                        Brevmal.VARSEL_OM_ÅRLIG_REVURDERING_EØS_MED_INNHENTING_AV_OPPLYSNINGER,
+                    ].includes(skjema.felter.brevmal.verdi) && (
+                        <StyledFamilieLandvelger
+                            erLesevisning={false}
+                            id={'mottakerlandSED'}
+                            label={'SED er sendt til'}
+                            kunEøs
+                            eksluderLand={['NO']}
```

Comment:
Har lagt til mulighet for å eksludere liste av land i FamilieLandvelger
<img width="351" alt="Skjermbilde 2022-11-03 kl  13 50 15" src="https://user-images.githubusercontent.com/58559732/199724997-6ccff2b9-d06e-4600-98ed-a6f0d0a08b60.png">